### PR TITLE
PHP Laravel web app 

### PR DESCRIPTION
**Pull request by dodaromike**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,88 @@
...
+page_type: sample
+description: This sample demonstrates how to use Microsoft Graph with a PHP web application and the Azure AD v2 authentication endpoint to access data in Office 365.
+products:
+- ms-graph
+- microsoft-graph-calendar-api
+- office-exchange-online
+languages:
+- php
+---
 # Microsoft Graph Training Module - Build PHP apps with Microsoft Graph
```
Comment (jasonjoh) [inline README.md]: Let's change the title here to `Microsoft Graph sample PHP Laravel web app`

File: README.md [hunk 2]
```diff
@@ -1,25 +1,88 @@
...
-## More samples
+1. Select **Azure Active Directory** in the left-hand navigation, then select **App registrations** under **Manage**.
 
-If you're looking for other Graph examples beyond calendar scenarios, or if you'd like to contribute other examples to this project, please see the [samples branch](https://github.com/microsoftgraph/msgraph-training-phpapp/tree/samples).
+1. Select **New registration**. On the **Register an application** page, set the values as follows.
 
-## Disclaimer
+    - Set **Name** to `PHP Graph Tutorial`.
+    - Set **Supported account types** to **Accounts in any organizational directory and personal Microsoft accounts**.
+    - Under **Redirect URI**, set the first drop-down to `Web` and set the value to `https://localhost:5001/`.
```
Comment (jasonjoh) [inline README.md]: This redirect URI isn't right.
```suggestion
    - Under **Redirect URI**, set the first drop-down to `Web` and set the value to `http://localhost:8000/callback`.
```

File: README.md [hunk 3]
```diff
@@ -1,25 +1,88 @@
...
+
+1. Select **Register**. On the **PHP Graph Tutorial** page, copy the value of the **Application (client) ID** and save it, you will need it in the next step.
+
+1. Select **Authentication** under **Manage**. Under **Redirect URIs** add a URI with the value `https://localhost:5001/signin-oidc`.
```
Comment (jasonjoh) [inline README.md]: Remove this step

File: README.md [hunk 4]
```diff
@@ -1,25 +1,88 @@
...
+
+1. Set the **Logout URL** to `https://localhost:5001/signout-oidc`.
```
Comment (jasonjoh) [inline README.md]: Remove this step

File: README.md [hunk 5]
```diff
@@ -1,25 +1,88 @@
...
+
+1. Locate the **Implicit grant** section and enable **ID tokens**. Select **Save**.
```
Comment (jasonjoh) [inline README.md]: Remove this step

**Review by jasonjoh — DISMISSED** [review]

**Comment by jasonjoh** [comment]
Thanks Mike! If you would, please delete the project in the demo folder and replace it with the project you generated. Want to be sure we capture the changes with the newer tools and dependencies.

**Comment by dodaromike** [comment]
I've made changes requested and followed the remaining instructions in the OneNote PHP Laravel web app.  A conflict is showing in demo/graph-tutorial/composer.lock, but Resolve conflicts isn't functional.

**Comment by dodaromike** [comment]
I've made changes requested and followed the remaining instructions in the OneNote file.  There are conflicts showing in demo/graph-tutorial/composer.lock that I can't resolve.

**Pull request merged**